Makefile

```
@@ -3,6 +3,7 @@
 IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 FLAGD_VERSION=v0.2.5
+CHART_VERSION=# x-release-please-version
```

```suggestion
CHART_VERSION=v0.2.6 # x-release-please-version
```
I think release-please needs something to replace

---

updated